starknet_py/utils/types.py

```diff
@@ -3,7 +3,7 @@
 # noinspection PyUnresolvedReferences
 # noinspection PyUnresolvedReferences
 from starknet_py.cairo.felt import decode_shortstring, encode_shortstring
-from starknet_py.net.models import Invoke, InvokeFunction, Transaction
+from starknet_py.net.models import Invoke, Transaction
 
 # noinspection PyUnresolvedReferences
 from starknet_py.net.models.address import parse_address
```

Comment:
I wonder if there's a point in keeping this file. It's supposed to be here for backwards compatibility but it must be compatibility with a really old version. 

Maybe let's deprecate it?

Reply:
In my opinion, it could be removed now. It was added almost a year ago
![image](https://user-images.githubusercontent.com/61014013/211768564-2305dad5-09fd-4544-968f-d8efde9b7db4.png)

Reply:
I'm not 100% sure if outright removal should be good, but a short deprecation (think one version of library) would be fine.

Reply:
Though we already remove a bunch of things from `client_models` so maybe it's fine to do this without deprecation. Not sure.

Reply:
It's a file with imports only. Users can reimport it without any problems